any_parser/any_parser.py

```diff
@@ -477,6 +488,8 @@ def async_fetch(
                 return result["json"]
             elif "resume_extraction" in result:
                 return result["resume_extraction"]
+            elif "pii_extraction" in result:
+                return result["pii_extraction"]
             elif "markdown" in result:
                 markdown_list = result["markdown"]
                 return "\n".join(markdown_list)
```

Comment:
update CDK to make sure each result has same key. If same key does not work, use its corresponding key as the api name.